lib/cequel/record/persistence.rb

```diff
@@ -82,7 +82,7 @@ def hydrate(row)
       # @since 1.0.0
       #
       def key_attributes
-        @attributes.slice(*self.class.key_column_names)
+        @cequel_attributes.slice(*self.class.key_column_names)
```

Comment:
why the name change?

Reply:
When `@attributes` is exposed in Rails 5.2 it causes problems

Reply:
What problems does it cause? See <https://github.com/cequel/cequel/pull/395#discussion_r226765188> for detailed discussion.